Makefile

```diff
@@ -0,0 +1,4 @@
+fetch:
+	pipenv run ./talkpython.py --download-dir talkpython.fm
+	pipenv run ./pythonbytes.py --download-dir pythonbytes.fm
+
```

Comment:
It might make sense to you, I don't want to introduce a Makefile just yet. Will see, but definitely not some fixed directory names.